platform/android/java_godot_wrapper.cpp

```diff
@@ -331,11 +331,14 @@ void GodotJavaWrapper::init_input_devices() {
 	}
 }
 
-void GodotJavaWrapper::vibrate(int p_duration_ms) {
+void GodotJavaWrapper::vibrate(int p_duration_ms, float p_amplitude) {
 	if (_vibrate) {
 		JNIEnv *env = get_jni_env();
 		ERR_FAIL_NULL(env);
-		env->CallVoidMethod(godot_instance, _vibrate, p_duration_ms);
+
+		p_amplitude = CLAMP(p_amplitude, 0.0, 1.0);
+
+		env->CallVoidMethod(godot_instance, _vibrate, p_duration_ms, int(p_amplitude * 255));
```

Comment:
This may introduce a bug and behavior change on Android:
- The corresponding Android api [states](https://developer.android.com/reference/android/os/VibrationEffect#createOneShot(long,%20int)) that the amplitude must be between 1 and 255; the range being passed here is between 0 and 255
- The default amplitude value for this method is now `1.0`, which will correspond to the highest amplitude value (`255`) on Android. Depending on the device this may be different from the current behavior which uses the [`DEFAULT_AMPLITUDE`](https://developer.android.com/reference/android/os/VibrationEffect#DEFAULT_AMPLITUDE).

Reply:
Maybe the default amplitude should be `-1.0` (and this would be the default parameter).